units/Cargo.toml

```diff
@@ -16,6 +16,7 @@ exclude = ["tests", "contrib"]
 default = ["std"]
 std = ["alloc", "internals/std"]
 alloc = ["internals/alloc"]
+arbitrary = ["dep:arbitrary"]
```

Comment:
I don't believe this does anything, we would only need this if the `arbitrary` feature was used to enable another feature (like how it does in `primitives`).

Reply:
Right, in bitcoin/Cargo.toml if the bitcoin crate is built with the arbitrary feature, the dependent crates also need to be built with the arbitrary feature flag.  I really think it would be better to remove this before merging since it makes the toml file more confusing, although I guess it's not dangerous per say.

Reply:
ill push a change to get rid of it